charts/catalog/values.yaml

```diff
@@ -1,12 +1,12 @@
 # Default values for Service Catalog
+# service-catalog image to use
+image: quay.io/kubernetes-service-catalog/service-catalog:v0.0.23
+# imagePullPolicy for the service-catalog; valid values are "IfNotPresent",
+# "Never", and "Always"
+imagePullPolicy: Always
```

Comment:
I wish we wouldn't default to this too.

Reply:
I would think that most of our users in production would want to always pull the latest. What are you suggesting as a better value? IfNotPresent?

Reply:
yes I prefer "IfNotPresent". I'm continually running into cases where I already have the image in question locally downloaded and then this flag forces it to be re-fetched, and in some cases pull down a version I didn't want, usually because I did a custom build before hand.  Plus, when I'm on a slow internet (like on a plane), refetching is a killer sometimes.

Reply:
My concern is that this makes it more convenient as a developer of the product but more problematic as a user of the product. If that is the case, then I would prefer to keep it as "Always". As a developer, it shouldn't be too bothersome to simply override the imagePullPolicy when running the chart.

Reply:
In any event, someone with a dog in this fight should create an issue for this so that we can have a discussion about it outside of this PR.